docs/source/references/configurations.rst

```diff
@@ -70,4 +70,49 @@ It only makes sense to reset the session when the backend database is changed si
 Thus, the only place to change the LineaPy database is at the beginning of the notebook.
 
 Note that, you need to make sure whenever you are setting `LINEAPY_DATABASE_URL`, you point to the  `LINEAPY_ARTIFACT_STORAGE_DIR`.
-If not, ``Artifact.get_value`` might return an error that is related cannot find underlying pickle object. 
\ No newline at end of file
+If not, ``Artifact.get_value`` might return an error that is related cannot find underlying pickle object.
+
+
+
+Artifact Storage Location
+=========================
+
+You can change the artifact storage location by setting the `LINEAPY_ARTIFACT_STORAGE_DIR` environmental variable, or other ways mentioned in the above session.
+LineaPy is using the IO handler from ``pandas`` directly.
+In theory, we can use various filesystems like S3, GCS, Azure, etc... to store the artifact, just like we are using ``pandas.DataFrame.to_csv`` to save CSV files.
+Note that, LineaPy only supports S3 at this moment.
+Some filesystems might need extra configuration items to be set, in ``pandas``, you can pass these configurations as ``storage_options`` in ``pandas.DataFrame.to_csv(storage_options={some storage options})``, where the `storage_options` is a filesystem-specific dictionary pass into `fsspec.filesystem`.  .
```

Comment:
add link to fsspec doc explaining the options/backends we can use. atleast the prefixes if not the whole thing.